x-pack/plugins/reporting/server/lib/tasks/monitor_reports.ts

```diff
@@ -126,33 +135,19 @@ export class MonitorReportsTask implements ReportingTask {
       createTaskRunner: this.getTaskRunner(),
       maxAttempts: 1,
       // round the timeout value up to the nearest second, since Task Manager
-      // doesn't support milliseconds
+      // doesn't support milliseconds or > 1s
       timeout: Math.ceil(this.timeout.asSeconds()) + 's',
     };
   }
 
-  // reschedule the task with TM and update the report document status to "Pending"
+  // reschedule the task with TM
   private async rescheduleTask(task: ReportTaskParams, logger: LevelLogger) {
     if (!this.taskManagerStart) {
       throw new Error('Reporting task runner has not been initialized!');
     }
-    logger.info(`Rescheduling ${task.id} to retry after timeout expiration.`);
-
-    const store = await this.getStore();
-
-    const oldTaskInstance: ReportingExecuteTaskInstance = {
-      taskType: REPORTING_EXECUTE_TYPE, // schedule a task to EXECUTE
-      state: {},
-      params: task,
-    };
-
-    const [report, newTask] = await Promise.all([
-      await store.findReportFromTask(task),
-      await this.taskManagerStart.schedule(oldTaskInstance),
-    ]);
-
-    await store.setReportPending(report);
```

Comment:
This PR adds a new method called `reportingStore.prepareReportForRetry` and it is called higher up in the code. It cuts out a couple of unnecessary queries here - we don't need to set the report status to `pending` because that was done at the same time the process_expiration was cleared.